### DISCUSSION TITLE
Request headers

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1232] [Q&A] [answered]
Can the request headers of the connection be viewed by the client sir?

**Comment by miguelgrinberg** [comment]
What request headers?

**Reply by Notadumdum** [reply]
The HTTP request headers sir.. origin,host and every other header regarding that particular connection

**Reply by miguelgrinberg** [reply, marked as answer]
No, you cannot access these headers, as this does not make much sense in the client. Origin is a browser header, so it does not apply here. The host you already know it, since you are passing it in the connection URL.

**Reply by Notadumdum** [reply]
Ok sir
